test/node-unit/cli/options.spec.js

```diff
@@ -199,6 +199,37 @@ describe('options', function () {
           });
         });
 
+        describe('when path to package.json unspecified and package.json exists but is invalid', function () {
+          beforeEach(function () {
+            const filepath = '/some/package.json';
+            readFileSync = sinon.stub();
+            // package.json
+            readFileSync
+              .onFirstCall()
+              // Note the extra comma
+              .returns('{"mocha": {"retries": 3, "_": ["foobar.spec.js"],}}');
```

Comment:
[Testing] Nit: no need to have a difficult-to-read JSON that needs a comment, when we can have something more straightforward:

```suggestion
              .returns('{definitely-invalid');
```

Reply:
Ah, yes, that is a lot clearer (the most common error people will make in JSON is obviously the trailing comma, but that isn't very relevant for this test case)